pkg/vpcmodel/nodesConnectivity.go

```diff
@@ -164,11 +187,8 @@ func (c *VPCConfig) getAllowedConnsPerDirection(isIngress bool, capturedNode Nod
 			}
 			allLayersRes[peerNode] = routerConnRes
 		}
-		loadBalancerRule := c.getLoadBalancerRule(src, dst)
-		if loadBalancerRule != nil && loadBalancerRule.Deny(isIngress) {
-			allLayersRes[peerNode] = NoConns()
-			continue
-		}
+		otherRulesConn := c.getOtherRulesConn(src, dst, isIngress)
```

Comment:
same comment regarding `other`

Reply:
here I replaces "other" with "more" to keep it short